src/Components/DataHistoryExplorer/Internal/TimeSeriesViewer/Internal/TimeSeriesTable/TimeSeriesTable.styles.ts

```diff
@@ -0,0 +1,49 @@
+import { FontWeights, mergeStyleSets } from '@fluentui/react';
+import {
+    ITimeSeriesTableStyleProps,
+    ITimeSeriesTableStyles
+} from './TimeSeriesTable.types';
+
+export const classPrefix = 'cb-time-series-table';
+const classNames = {
+    root: `${classPrefix}-root`,
+    listWrapper: `${classPrefix}-list-wrapper`,
+    notSetCell: `${classPrefix}-not-set-cell`
+};
+export const getStyles = (
+    _props: ITimeSeriesTableStyleProps
+): ITimeSeriesTableStyles => {
+    return {
+        root: [
+            classNames.root,
+            {
+                width: '100%',
+                height: '100%',
+                display: 'flex',
+                flexDirection: 'column'
+            }
+        ],
+        listWrapper: [
+            classNames.listWrapper,
+            { overflow: 'auto', position: 'relative', flexGrow: 1 }
+        ],
+        notSetCell: [classNames.notSetCell, { fontStyle: 'italic' }],
+        subComponentStyles: {
+            loadingSpinner: { root: { height: '100%' } },
+            detailsList: {
+                root: { overflow: 'hidden', '.ms-DetailsRow': { fontSize: 13 } }
+            },
+            seriesColumn: (props) => {
+                return mergeStyleSets({
```

Comment:
why do you need mergeStyleSets here?

Reply:
I need to return a IProcessedStyleSet here to access its **root** what assigning it as a className as you also used in other places like [here](https://github.com/microsoft/iot-cardboard-js/blob/800150615c09ea53410ffcd1c1b0ec4d004db2e6/src/Components/ColorPills/ColorPills.types.ts#L46)